spec/models/waste_exemptions_engine/transient_address_spec.rb

```diff
@@ -3,7 +3,7 @@
 require "rails_helper"
 
 module WasteExemptionsEngine
-  RSpec.describe TransientAddress, type: :model do
+  RSpec.describe TransientAddress, type: :model, vcr: true do
```

Comment:
Can we move `vcr: true` to just the relevant context?

Reply:
NP. That was me quickly looking at another spec and assuming it needed to be at the top!